rocketpy/EnvironmentAnalysis.py

```diff
@@ -605,12 +606,84 @@ def calculate_average_wind_profile(self):
         """average, 1, 2, 3 sigma wind profile from 0 35,000 ft AGL"""
         ...
 
-    # TODO: Implement
-    def calculate_average_day_wind_rose(self):
-        """average day wind rose"""
-        ...
+    def animate_average_wind_rose(self, figsize=(8, 8)):
+        """Animates the wind_rose of an average day. The inputs of a wind_rose are the location of the
+        place where we want to analyse, (x,y,z). The data is ensembled by hour, which means, the windrose
+        of a specific hour is generated by bringing together the data of all of the days available for that
+        specific hour. It's possible to change the size of the gif using the parameter figsize, which is the
+        height and width in inches.
+
+        Parameters
+        ----------
+        figsize : array
+
+        Returns
+        -------
+        Image : ipywidgets.widgets.widget_media.Image
+        """
+        days = list(self.surfaceDataDict.keys())
+        hours = list(self.surfaceDataDict[days[0]].keys())
+
+        windSpeed = {}
+        windDir = {}
+
+        for hour in hours:
+            windSpeed[hour] = []
+            windDir[hour] = []
+            for day in days:
+                windSpeed[hour].append(
+                    self.pressureLevelDataDict[day][hour]["windSpeed"](self.elevation)
+                )
+                windDir[hour].append(
+                    self.pressureLevelDataDict[day][hour]["windDirection"](
+                        self.elevation
+                    )
+                )
+
+        fig = plt.figure(
+            facecolor="w", edgecolor="w", figsize=figsize
+        )  # initialise la figure
+
+        filename = "wind_rose.gif"
+        FFMpegWriter = matplotlib.animation.writers["ffmpeg"]
+        metadata = dict(
+            title="windrose",
+            artist="windrose",
+            comment="""Made with windrose
+                http://www.github.com/scls19fr/windrose""",
+        )
+        writer = FFMpegWriter(fps=1, metadata=metadata)
+
+        with writer.saving(fig, filename, 100):
+            for hour in hours:
+                ax = WindroseAxes.from_ax(fig=fig)
```

Comment:
Would it be possible to fix the radial axis (in this case: 0, 4.5, 9.0, 13.5, 18, ...) for every frame?

![image](https://user-images.githubusercontent.com/572483/167684840-a4b861e0-3f94-466c-91cd-644eafe32d43.png)